lib/iris/config.py

```diff
@@ -132,8 +132,12 @@ def get_dir_option(section, option, default=None):
 # Override the data repository if the appropriate environment variable
 # has been set.  This is used in setup.py in the TestRunner command to
 # enable us to simulate the absence of external data.
-if os.environ.get("override_test_data_repository"):
-    TEST_DATA_DIR = None
+override = os.environ.get("override_test_data_repository")
+if override:
+    if override == '1':
```

Comment:
then the test data directory will not be found, and the tests will fall back to the 'no data' approach of skipping